servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/KeepAliveManagerTest.java

```diff
@@ -47,41 +59,54 @@
 
 class KeepAliveManagerTest {
 
+    private final BlockingQueue<ScheduledTask> scheduledTasks = new LinkedBlockingQueue<>();
```

Comment:
I know it wasn't done in the original code / this refactoring, but would it make sense in the BeforeEach method (or setUp) to clear this queue to have every test start with a deterministic fresh state?